### PR TITLE
Feature bl 5235 creating audit table for unsubscriptions

### PR DESCRIPTION
**Pull request by Thomas-Franklin**

### CODE REVIEW

File: webapp/src/test/java/uk/gov/dvsa/motr/web/component/subscription/service/SubscriptionConfirmationServiceTest.java [hunk 1]
```diff
@@ -40,7 +40,6 @@
 
     @Before
     public void setUp() {
-
```
Comment (aaronlong) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/component/subscription/service/SubscriptionConfirmationServiceTest.java]: Following the coding standards there should be a blank line at the beginning of all these methods

File: webapp/src/test/java/uk/gov/dvsa/motr/web/component/subscription/service/UnsubscribeServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+        this.unsubscriptionService = new UnsubscribeService(
+                subscriptionRepository,
+                cancelledSubscriptionRepository
+        );
+    }
+
+    @Test
+    public void unsubscribeWhenSubscriptionDoesExist() throws Exception {
+        Subscription subscription = subscriptionStub();
```
Comment (aaronlong) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/component/subscription/service/UnsubscribeServiceTest.java]: Please make sure all new methods have the empty line at the start of the method :)

File: webapp/src/integration-test/java/uk/gov/dvsa/motr/test/integration/dynamodb/fixture/model/CancelledSubscriptionItem.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package uk.gov.dvsa.motr.test.integration.dynamodb.fixture.model;
+
+import uk.gov.dvsa.motr.test.data.RandomDataUtil;
+
+import java.util.UUID;
+
+public class CancelledSubscriptionItem {
+
+    private String unsubscribeId = UUID.randomUUID().toString();
+
```
Comment (cormacq) [inline webapp/src/integration-test/java/uk/gov/dvsa/motr/test/integration/dynamodb/fixture/model/CancelledSubscriptionItem.java]: Do we have a style guide for adding spaces after declaring/initialising variables? Sometimes there is spaces, sometimes there isn't. Happy with either as long as it's consistent :)

Comment (Thomas-Franklin) [inline webapp/src/integration-test/java/uk/gov/dvsa/motr/test/integration/dynamodb/fixture/model/CancelledSubscriptionItem.java]: The style guide doesn't have any rules for this. Typically I stick to grouping if it is I who wrote it, i.e. private types grouped together, private statics grouped, etc. 

Of course this isn't applied here - I can change based on consensus.

**Pull request merged**